eng/config/PublishData.json

```diff
@@ -44,7 +44,6 @@
       "Microsoft.CodeAnalysis.CSharp.EditorFeatures": "vssdk",
       "Microsoft.CodeAnalysis.InteractiveHost": "vssdk",
       "Microsoft.CodeAnalysis.VisualBasic.EditorFeatures": "vssdk",
-      "Microsoft.CodeAnalysis.EditorFeatures.Wpf": "vssdk",
```

Comment:
This needs to stay around for servicing, since the file in main is used for servicing branches.

Reply:
can you clarify?  just so i understand?

Reply:
The main branch PublishData.json is the source of truth for all builds. @dibarbet Recently suggested that release/* branches should simply refer to their own copy.